### ISSUE TITLE
The diff between original yolo2 and after add focal-loss?

### ISSUE THREAD

**Issue by DanLiu0623** [marvis/pytorch-yolo2#71]

**Comment by alphacoco** [comment]
I guess u should use: `FL = FocalLoss(class_num=5, size_average=False)`. 'cause the `size_average=False` is in the `nn. CrossEntropyLoss`
